19 окт. 2025 13:55:37,286 ERROR SystemLogger - Failed to initialize tess4j. Add tess4j jar to classpath.
19 окт. 2025 14:13:07,211 ERROR SystemLogger - Failed to initialize tess4j. Add tess4j jar to classpath.
19 окт. 2025 14:31:50,813 ERROR LogicsModule - java.net.ConnectException Connection refused: connect at '19.10.2025, 14:31' from 'dilpkn': 

	at java.base/sun.nio.ch.Net.connect0(Native Method)
	at java.base/sun.nio.ch.Net.connect(Net.java:579)
	at java.base/sun.nio.ch.Net.connect(Net.java:568)
	at java.base/sun.nio.ch.NioSocketImpl.connect(NioSocketImpl.java:585)
	at java.base/java.net.SocksSocketImpl.connect(SocksSocketImpl.java:327)
	at java.base/java.net.Socket.connect(Socket.java:633)
	at java.base/java.net.Socket.connect(Socket.java:583)
	at java.base/java.net.Socket.<init>(Socket.java:507)
	at java.base/java.net.Socket.<init>(Socket.java:287)
	at lsfusion.base.remote.ZipSocket.<init>(ZipSocket.java:26)
	at lsfusion.base.remote.ZipClientSocketFactory.createSocket(ZipClientSocketFactory.java:34)
	at java.rmi/sun.rmi.transport.tcp.TCPEndpoint.newSocket(TCPEndpoint.java:620)
	at java.rmi/sun.rmi.transport.tcp.TCPChannel.createConnection(TCPChannel.java:217)
	at java.rmi/sun.rmi.transport.tcp.TCPChannel.newConnection(TCPChannel.java:204)
	at java.rmi/sun.rmi.server.UnicastRef.invoke(UnicastRef.java:134)
	at java.rmi/java.rmi.server.RemoteObjectInvocationHandler.invokeRemoteMethod(RemoteObjectInvocationHandler.java:215)
	at java.rmi/java.rmi.server.RemoteObjectInvocationHandler.invoke(RemoteObjectInvocationHandler.java:160)
	at jdk.proxy2/jdk.proxy2.$Proxy13.pullMessages(Unknown Source)
	at lsfusion.client.controller.remote.PingThread$1.call(PingThread.java:70)
	at lsfusion.client.controller.remote.PingThread$1.call(PingThread.java:1)
	at lsfusion.client.controller.remote.RmiQueue.runRetryableRequest(RmiQueue.java:148)
	at lsfusion.client.controller.remote.RmiQueue.runRetryableRequest(RmiQueue.java:101)
	at lsfusion.client.controller.remote.RmiQueue.runRetryableRequest(RmiQueue.java:97)
	at lsfusion.client.controller.remote.PingThread.run(PingThread.java:68)


19 окт. 2025 14:31:50,875 ERROR LogicsModule - java.net.ConnectException Connection refused: connect at '19.10.2025, 14:31' from 'dilpkn': 

	at java.base/sun.nio.ch.Net.connect0(Native Method)
	at java.base/sun.nio.ch.Net.connect(Net.java:579)
	at java.base/sun.nio.ch.Net.connect(Net.java:568)
	at java.base/sun.nio.ch.NioSocketImpl.connect(NioSocketImpl.java:585)
	at java.base/java.net.SocksSocketImpl.connect(SocksSocketImpl.java:327)
	at java.base/java.net.Socket.connect(Socket.java:633)
	at java.base/java.net.Socket.connect(Socket.java:583)
	at java.base/java.net.Socket.<init>(Socket.java:507)
	at java.base/java.net.Socket.<init>(Socket.java:287)
	at lsfusion.base.remote.ZipSocket.<init>(ZipSocket.java:26)
	at lsfusion.base.remote.ZipClientSocketFactory.createSocket(ZipClientSocketFactory.java:34)
	at java.rmi/sun.rmi.transport.tcp.TCPEndpoint.newSocket(TCPEndpoint.java:620)
	at java.rmi/sun.rmi.transport.tcp.TCPChannel.createConnection(TCPChannel.java:217)
	at java.rmi/sun.rmi.transport.tcp.TCPChannel.newConnection(TCPChannel.java:204)
	at java.rmi/sun.rmi.server.UnicastRef.invoke(UnicastRef.java:134)
	at java.rmi/java.rmi.server.RemoteObjectInvocationHandler.invokeRemoteMethod(RemoteObjectInvocationHandler.java:215)
	at java.rmi/java.rmi.server.RemoteObjectInvocationHandler.invoke(RemoteObjectInvocationHandler.java:160)
	at jdk.proxy2/jdk.proxy2.$Proxy13.pullMessages(Unknown Source)
	at lsfusion.client.controller.remote.PingThread$1.call(PingThread.java:70)
	at lsfusion.client.controller.remote.PingThread$1.call(PingThread.java:1)
	at lsfusion.client.controller.remote.RmiQueue.runRetryableRequest(RmiQueue.java:148)
	at lsfusion.client.controller.remote.RmiQueue.runRetryableRequest(RmiQueue.java:101)
	at lsfusion.client.controller.remote.RmiQueue.runRetryableRequest(RmiQueue.java:97)
	at lsfusion.client.controller.remote.PingThread.run(PingThread.java:68)


19 окт. 2025 14:31:50,911 ERROR LogicsModule - java.net.ConnectException Connection refused: connect at '19.10.2025, 14:31' from 'dilpkn': 

	at java.base/sun.nio.ch.Net.connect0(Native Method)
	at java.base/sun.nio.ch.Net.connect(Net.java:579)
	at java.base/sun.nio.ch.Net.connect(Net.java:568)
	at java.base/sun.nio.ch.NioSocketImpl.connect(NioSocketImpl.java:585)
	at java.base/java.net.SocksSocketImpl.connect(SocksSocketImpl.java:327)
	at java.base/java.net.Socket.connect(Socket.java:633)
	at java.base/java.net.Socket.connect(Socket.java:583)
	at java.base/java.net.Socket.<init>(Socket.java:507)
	at java.base/java.net.Socket.<init>(Socket.java:287)
	at lsfusion.base.remote.ZipSocket.<init>(ZipSocket.java:26)
	at lsfusion.base.remote.ZipClientSocketFactory.createSocket(ZipClientSocketFactory.java:34)
	at java.rmi/sun.rmi.transport.tcp.TCPEndpoint.newSocket(TCPEndpoint.java:620)
	at java.rmi/sun.rmi.transport.tcp.TCPChannel.createConnection(TCPChannel.java:217)
	at java.rmi/sun.rmi.transport.tcp.TCPChannel.newConnection(TCPChannel.java:204)
	at java.rmi/sun.rmi.server.UnicastRef.invoke(UnicastRef.java:134)
	at java.rmi/java.rmi.server.RemoteObjectInvocationHandler.invokeRemoteMethod(RemoteObjectInvocationHandler.java:215)
	at java.rmi/java.rmi.server.RemoteObjectInvocationHandler.invoke(RemoteObjectInvocationHandler.java:160)
	at jdk.proxy2/jdk.proxy2.$Proxy4.createNavigator(Unknown Source)
	at lsfusion.client.controller.remote.proxy.RemoteLogicsProxy.createNavigator_aroundBody0(RemoteLogicsProxy.java:28)
	at lsfusion.client.controller.remote.proxy.RemoteLogicsProxy.createNavigator_aroundBody1$advice(RemoteLogicsProxy.java:86)
	at lsfusion.client.controller.remote.proxy.RemoteLogicsProxy.createNavigator(RemoteLogicsProxy.java:1)
	at lsfusion.client.view.MainFrame$1$1.lambda$0(MainFrame.java:79)
	at lsfusion.interop.logics.AbstractLogicsProviderImpl.runRequest(AbstractLogicsProviderImpl.java:70)
	at lsfusion.interop.logics.AbstractLogicsProviderImpl.runRequest(AbstractLogicsProviderImpl.java:64)
	at lsfusion.client.controller.MainController.runRequest(MainController.java:446)
	at lsfusion.client.view.MainFrame$1$1.call(MainFrame.java:74)
	at lsfusion.client.view.MainFrame$1$1.call(MainFrame.java:1)
	at lsfusion.client.controller.remote.RmiQueue.runRetryableRequest(RmiQueue.java:148)
	at lsfusion.client.controller.remote.RmiQueue.runRetryableRequest(RmiQueue.java:101)
	at lsfusion.client.controller.remote.RmiQueue.runRetryableRequest(RmiQueue.java:97)
	at lsfusion.client.view.MainFrame$1.call(MainFrame.java:71)
	at lsfusion.client.view.MainFrame$1.call(MainFrame.java:1)
	at lsfusion.client.controller.remote.ReconnectWorker.doInBackground(ReconnectWorker.java:33)
	at java.desktop/javax.swing.SwingWorker$1.call(SwingWorker.java:304)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.desktop/javax.swing.SwingWorker.run(SwingWorker.java:343)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:833)


19 окт. 2025 14:31:54,283 ERROR SystemLogger - Failed to initialize tess4j. Add tess4j jar to classpath.
19 окт. 2025 14:33:07,625 ERROR LogicsModule - java.net.ConnectException Connection refused: connect at '19.10.2025, 14:33' from 'dilpkn': 

	at java.base/sun.nio.ch.Net.connect0(Native Method)
	at java.base/sun.nio.ch.Net.connect(Net.java:579)
	at java.base/sun.nio.ch.Net.connect(Net.java:568)
	at java.base/sun.nio.ch.NioSocketImpl.connect(NioSocketImpl.java:585)
	at java.base/java.net.SocksSocketImpl.connect(SocksSocketImpl.java:327)
	at java.base/java.net.Socket.connect(Socket.java:633)
	at java.base/java.net.Socket.connect(Socket.java:583)
	at java.base/java.net.Socket.<init>(Socket.java:507)
	at java.base/java.net.Socket.<init>(Socket.java:287)
	at lsfusion.base.remote.ZipSocket.<init>(ZipSocket.java:26)
	at lsfusion.base.remote.ZipClientSocketFactory.createSocket(ZipClientSocketFactory.java:34)
	at java.rmi/sun.rmi.transport.tcp.TCPEndpoint.newSocket(TCPEndpoint.java:620)
	at java.rmi/sun.rmi.transport.tcp.TCPChannel.createConnection(TCPChannel.java:217)
	at java.rmi/sun.rmi.transport.tcp.TCPChannel.newConnection(TCPChannel.java:204)
	at java.rmi/sun.rmi.server.UnicastRef.invoke(UnicastRef.java:134)
	at java.rmi/java.rmi.server.RemoteObjectInvocationHandler.invokeRemoteMethod(RemoteObjectInvocationHandler.java:215)
	at java.rmi/java.rmi.server.RemoteObjectInvocationHandler.invoke(RemoteObjectInvocationHandler.java:160)
	at jdk.proxy2/jdk.proxy2.$Proxy13.pullMessages(Unknown Source)
	at lsfusion.client.controller.remote.PingThread$1.call(PingThread.java:70)
	at lsfusion.client.controller.remote.PingThread$1.call(PingThread.java:1)
	at lsfusion.client.controller.remote.RmiQueue.runRetryableRequest(RmiQueue.java:148)
	at lsfusion.client.controller.remote.RmiQueue.runRetryableRequest(RmiQueue.java:101)
	at lsfusion.client.controller.remote.RmiQueue.runRetryableRequest(RmiQueue.java:97)
	at lsfusion.client.controller.remote.PingThread.run(PingThread.java:68)


19 окт. 2025 14:33:07,692 ERROR LogicsModule - java.net.ConnectException Connection refused: connect at '19.10.2025, 14:33' from 'dilpkn': 

	at java.base/sun.nio.ch.Net.connect0(Native Method)
	at java.base/sun.nio.ch.Net.connect(Net.java:579)
	at java.base/sun.nio.ch.Net.connect(Net.java:568)
	at java.base/sun.nio.ch.NioSocketImpl.connect(NioSocketImpl.java:585)
	at java.base/java.net.SocksSocketImpl.connect(SocksSocketImpl.java:327)
	at java.base/java.net.Socket.connect(Socket.java:633)
	at java.base/java.net.Socket.connect(Socket.java:583)
	at java.base/java.net.Socket.<init>(Socket.java:507)
	at java.base/java.net.Socket.<init>(Socket.java:287)
	at lsfusion.base.remote.ZipSocket.<init>(ZipSocket.java:26)
	at lsfusion.base.remote.ZipClientSocketFactory.createSocket(ZipClientSocketFactory.java:34)
	at java.rmi/sun.rmi.transport.tcp.TCPEndpoint.newSocket(TCPEndpoint.java:620)
	at java.rmi/sun.rmi.transport.tcp.TCPChannel.createConnection(TCPChannel.java:217)
	at java.rmi/sun.rmi.transport.tcp.TCPChannel.newConnection(TCPChannel.java:204)
	at java.rmi/sun.rmi.server.UnicastRef.invoke(UnicastRef.java:134)
	at java.rmi/java.rmi.server.RemoteObjectInvocationHandler.invokeRemoteMethod(RemoteObjectInvocationHandler.java:215)
	at java.rmi/java.rmi.server.RemoteObjectInvocationHandler.invoke(RemoteObjectInvocationHandler.java:160)
	at jdk.proxy2/jdk.proxy2.$Proxy13.pullMessages(Unknown Source)
	at lsfusion.client.controller.remote.PingThread$1.call(PingThread.java:70)
	at lsfusion.client.controller.remote.PingThread$1.call(PingThread.java:1)
	at lsfusion.client.controller.remote.RmiQueue.runRetryableRequest(RmiQueue.java:148)
	at lsfusion.client.controller.remote.RmiQueue.runRetryableRequest(RmiQueue.java:101)
	at lsfusion.client.controller.remote.RmiQueue.runRetryableRequest(RmiQueue.java:97)
	at lsfusion.client.controller.remote.PingThread.run(PingThread.java:68)


19 окт. 2025 14:33:07,724 ERROR LogicsModule - java.net.ConnectException Connection refused: connect at '19.10.2025, 14:33' from 'dilpkn': 

	at java.base/sun.nio.ch.Net.connect0(Native Method)
	at java.base/sun.nio.ch.Net.connect(Net.java:579)
	at java.base/sun.nio.ch.Net.connect(Net.java:568)
	at java.base/sun.nio.ch.NioSocketImpl.connect(NioSocketImpl.java:585)
	at java.base/java.net.SocksSocketImpl.connect(SocksSocketImpl.java:327)
	at java.base/java.net.Socket.connect(Socket.java:633)
	at java.base/java.net.Socket.connect(Socket.java:583)
	at java.base/java.net.Socket.<init>(Socket.java:507)
	at java.base/java.net.Socket.<init>(Socket.java:287)
	at lsfusion.base.remote.ZipSocket.<init>(ZipSocket.java:26)
	at lsfusion.base.remote.ZipClientSocketFactory.createSocket(ZipClientSocketFactory.java:34)
	at java.rmi/sun.rmi.transport.tcp.TCPEndpoint.newSocket(TCPEndpoint.java:620)
	at java.rmi/sun.rmi.transport.tcp.TCPChannel.createConnection(TCPChannel.java:217)
	at java.rmi/sun.rmi.transport.tcp.TCPChannel.newConnection(TCPChannel.java:204)
	at java.rmi/sun.rmi.server.UnicastRef.invoke(UnicastRef.java:134)
	at java.rmi/java.rmi.server.RemoteObjectInvocationHandler.invokeRemoteMethod(RemoteObjectInvocationHandler.java:215)
	at java.rmi/java.rmi.server.RemoteObjectInvocationHandler.invoke(RemoteObjectInvocationHandler.java:160)
	at jdk.proxy2/jdk.proxy2.$Proxy4.createNavigator(Unknown Source)
	at lsfusion.client.controller.remote.proxy.RemoteLogicsProxy.createNavigator_aroundBody0(RemoteLogicsProxy.java:28)
	at lsfusion.client.controller.remote.proxy.RemoteLogicsProxy.createNavigator_aroundBody1$advice(RemoteLogicsProxy.java:86)
	at lsfusion.client.controller.remote.proxy.RemoteLogicsProxy.createNavigator(RemoteLogicsProxy.java:1)
	at lsfusion.client.view.MainFrame$1$1.lambda$0(MainFrame.java:79)
	at lsfusion.interop.logics.AbstractLogicsProviderImpl.runRequest(AbstractLogicsProviderImpl.java:70)
	at lsfusion.interop.logics.AbstractLogicsProviderImpl.runRequest(AbstractLogicsProviderImpl.java:64)
	at lsfusion.client.controller.MainController.runRequest(MainController.java:446)
	at lsfusion.client.view.MainFrame$1$1.call(MainFrame.java:74)
	at lsfusion.client.view.MainFrame$1$1.call(MainFrame.java:1)
	at lsfusion.client.controller.remote.RmiQueue.runRetryableRequest(RmiQueue.java:148)
	at lsfusion.client.controller.remote.RmiQueue.runRetryableRequest(RmiQueue.java:101)
	at lsfusion.client.controller.remote.RmiQueue.runRetryableRequest(RmiQueue.java:97)
	at lsfusion.client.view.MainFrame$1.call(MainFrame.java:71)
	at lsfusion.client.view.MainFrame$1.call(MainFrame.java:1)
	at lsfusion.client.controller.remote.ReconnectWorker.doInBackground(ReconnectWorker.java:33)
	at java.desktop/javax.swing.SwingWorker$1.call(SwingWorker.java:304)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.desktop/javax.swing.SwingWorker.run(SwingWorker.java:343)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:833)


19 окт. 2025 14:33:11,773 ERROR SystemLogger - Failed to initialize tess4j. Add tess4j jar to classpath.
19 окт. 2025 14:33:57,976 ERROR LogicsModule - java.net.ConnectException Connection refused: connect at '19.10.2025, 14:33' from 'dilpkn': 

	at java.base/sun.nio.ch.Net.connect0(Native Method)
	at java.base/sun.nio.ch.Net.connect(Net.java:579)
	at java.base/sun.nio.ch.Net.connect(Net.java:568)
	at java.base/sun.nio.ch.NioSocketImpl.connect(NioSocketImpl.java:585)
	at java.base/java.net.SocksSocketImpl.connect(SocksSocketImpl.java:327)
	at java.base/java.net.Socket.connect(Socket.java:633)
	at java.base/java.net.Socket.connect(Socket.java:583)
	at java.base/java.net.Socket.<init>(Socket.java:507)
	at java.base/java.net.Socket.<init>(Socket.java:287)
	at lsfusion.base.remote.ZipSocket.<init>(ZipSocket.java:26)
	at lsfusion.base.remote.ZipClientSocketFactory.createSocket(ZipClientSocketFactory.java:34)
	at java.rmi/sun.rmi.transport.tcp.TCPEndpoint.newSocket(TCPEndpoint.java:620)
	at java.rmi/sun.rmi.transport.tcp.TCPChannel.createConnection(TCPChannel.java:217)
	at java.rmi/sun.rmi.transport.tcp.TCPChannel.newConnection(TCPChannel.java:204)
	at java.rmi/sun.rmi.server.UnicastRef.invoke(UnicastRef.java:134)
	at java.rmi/java.rmi.server.RemoteObjectInvocationHandler.invokeRemoteMethod(RemoteObjectInvocationHandler.java:215)
	at java.rmi/java.rmi.server.RemoteObjectInvocationHandler.invoke(RemoteObjectInvocationHandler.java:160)
	at jdk.proxy2/jdk.proxy2.$Proxy13.pullMessages(Unknown Source)
	at lsfusion.client.controller.remote.PingThread$1.call(PingThread.java:70)
	at lsfusion.client.controller.remote.PingThread$1.call(PingThread.java:1)
	at lsfusion.client.controller.remote.RmiQueue.runRetryableRequest(RmiQueue.java:148)
	at lsfusion.client.controller.remote.RmiQueue.runRetryableRequest(RmiQueue.java:101)
	at lsfusion.client.controller.remote.RmiQueue.runRetryableRequest(RmiQueue.java:97)
	at lsfusion.client.controller.remote.PingThread.run(PingThread.java:68)


19 окт. 2025 14:33:58,030 ERROR LogicsModule - java.net.ConnectException Connection refused: connect at '19.10.2025, 14:33' from 'dilpkn': 

	at java.base/sun.nio.ch.Net.connect0(Native Method)
	at java.base/sun.nio.ch.Net.connect(Net.java:579)
	at java.base/sun.nio.ch.Net.connect(Net.java:568)
	at java.base/sun.nio.ch.NioSocketImpl.connect(NioSocketImpl.java:585)
	at java.base/java.net.SocksSocketImpl.connect(SocksSocketImpl.java:327)
	at java.base/java.net.Socket.connect(Socket.java:633)
	at java.base/java.net.Socket.connect(Socket.java:583)
	at java.base/java.net.Socket.<init>(Socket.java:507)
	at java.base/java.net.Socket.<init>(Socket.java:287)
	at lsfusion.base.remote.ZipSocket.<init>(ZipSocket.java:26)
	at lsfusion.base.remote.ZipClientSocketFactory.createSocket(ZipClientSocketFactory.java:34)
	at java.rmi/sun.rmi.transport.tcp.TCPEndpoint.newSocket(TCPEndpoint.java:620)
	at java.rmi/sun.rmi.transport.tcp.TCPChannel.createConnection(TCPChannel.java:217)
	at java.rmi/sun.rmi.transport.tcp.TCPChannel.newConnection(TCPChannel.java:204)
	at java.rmi/sun.rmi.server.UnicastRef.invoke(UnicastRef.java:134)
	at java.rmi/java.rmi.server.RemoteObjectInvocationHandler.invokeRemoteMethod(RemoteObjectInvocationHandler.java:215)
	at java.rmi/java.rmi.server.RemoteObjectInvocationHandler.invoke(RemoteObjectInvocationHandler.java:160)
	at jdk.proxy2/jdk.proxy2.$Proxy13.pullMessages(Unknown Source)
	at lsfusion.client.controller.remote.PingThread$1.call(PingThread.java:70)
	at lsfusion.client.controller.remote.PingThread$1.call(PingThread.java:1)
	at lsfusion.client.controller.remote.RmiQueue.runRetryableRequest(RmiQueue.java:148)
	at lsfusion.client.controller.remote.RmiQueue.runRetryableRequest(RmiQueue.java:101)
	at lsfusion.client.controller.remote.RmiQueue.runRetryableRequest(RmiQueue.java:97)
	at lsfusion.client.controller.remote.PingThread.run(PingThread.java:68)


19 окт. 2025 14:33:58,053 ERROR LogicsModule - java.net.ConnectException Connection refused: connect at '19.10.2025, 14:33' from 'dilpkn': 

	at java.base/sun.nio.ch.Net.connect0(Native Method)
	at java.base/sun.nio.ch.Net.connect(Net.java:579)
	at java.base/sun.nio.ch.Net.connect(Net.java:568)
	at java.base/sun.nio.ch.NioSocketImpl.connect(NioSocketImpl.java:585)
	at java.base/java.net.SocksSocketImpl.connect(SocksSocketImpl.java:327)
	at java.base/java.net.Socket.connect(Socket.java:633)
	at java.base/java.net.Socket.connect(Socket.java:583)
	at java.base/java.net.Socket.<init>(Socket.java:507)
	at java.base/java.net.Socket.<init>(Socket.java:287)
	at lsfusion.base.remote.ZipSocket.<init>(ZipSocket.java:26)
	at lsfusion.base.remote.ZipClientSocketFactory.createSocket(ZipClientSocketFactory.java:34)
	at java.rmi/sun.rmi.transport.tcp.TCPEndpoint.newSocket(TCPEndpoint.java:620)
	at java.rmi/sun.rmi.transport.tcp.TCPChannel.createConnection(TCPChannel.java:217)
	at java.rmi/sun.rmi.transport.tcp.TCPChannel.newConnection(TCPChannel.java:204)
	at java.rmi/sun.rmi.server.UnicastRef.invoke(UnicastRef.java:134)
	at java.rmi/java.rmi.server.RemoteObjectInvocationHandler.invokeRemoteMethod(RemoteObjectInvocationHandler.java:215)
	at java.rmi/java.rmi.server.RemoteObjectInvocationHandler.invoke(RemoteObjectInvocationHandler.java:160)
	at jdk.proxy2/jdk.proxy2.$Proxy4.createNavigator(Unknown Source)
	at lsfusion.client.controller.remote.proxy.RemoteLogicsProxy.createNavigator_aroundBody0(RemoteLogicsProxy.java:28)
	at lsfusion.client.controller.remote.proxy.RemoteLogicsProxy.createNavigator_aroundBody1$advice(RemoteLogicsProxy.java:86)
	at lsfusion.client.controller.remote.proxy.RemoteLogicsProxy.createNavigator(RemoteLogicsProxy.java:1)
	at lsfusion.client.view.MainFrame$1$1.lambda$0(MainFrame.java:79)
	at lsfusion.interop.logics.AbstractLogicsProviderImpl.runRequest(AbstractLogicsProviderImpl.java:70)
	at lsfusion.interop.logics.AbstractLogicsProviderImpl.runRequest(AbstractLogicsProviderImpl.java:64)
	at lsfusion.client.controller.MainController.runRequest(MainController.java:446)
	at lsfusion.client.view.MainFrame$1$1.call(MainFrame.java:74)
	at lsfusion.client.view.MainFrame$1$1.call(MainFrame.java:1)
	at lsfusion.client.controller.remote.RmiQueue.runRetryableRequest(RmiQueue.java:148)
	at lsfusion.client.controller.remote.RmiQueue.runRetryableRequest(RmiQueue.java:101)
	at lsfusion.client.controller.remote.RmiQueue.runRetryableRequest(RmiQueue.java:97)
	at lsfusion.client.view.MainFrame$1.call(MainFrame.java:71)
	at lsfusion.client.view.MainFrame$1.call(MainFrame.java:1)
	at lsfusion.client.controller.remote.ReconnectWorker.doInBackground(ReconnectWorker.java:33)
	at java.desktop/javax.swing.SwingWorker$1.call(SwingWorker.java:304)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.desktop/javax.swing.SwingWorker.run(SwingWorker.java:343)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:833)


19 окт. 2025 14:34:00,600 ERROR SystemLogger - Failed to initialize tess4j. Add tess4j jar to classpath.
19 окт. 2025 16:49:09,670 ERROR LogicsModule - java.net.ConnectException Connection refused: connect at '19.10.2025, 16:49' from 'dilpkn': 

	at java.base/sun.nio.ch.Net.connect0(Native Method)
	at java.base/sun.nio.ch.Net.connect(Net.java:579)
	at java.base/sun.nio.ch.Net.connect(Net.java:568)
	at java.base/sun.nio.ch.NioSocketImpl.connect(NioSocketImpl.java:585)
	at java.base/java.net.SocksSocketImpl.connect(SocksSocketImpl.java:327)
	at java.base/java.net.Socket.connect(Socket.java:633)
	at java.base/java.net.Socket.connect(Socket.java:583)
	at java.base/java.net.Socket.<init>(Socket.java:507)
	at java.base/java.net.Socket.<init>(Socket.java:287)
	at lsfusion.base.remote.ZipSocket.<init>(ZipSocket.java:26)
	at lsfusion.base.remote.ZipClientSocketFactory.createSocket(ZipClientSocketFactory.java:34)
	at java.rmi/sun.rmi.transport.tcp.TCPEndpoint.newSocket(TCPEndpoint.java:620)
	at java.rmi/sun.rmi.transport.tcp.TCPChannel.createConnection(TCPChannel.java:217)
	at java.rmi/sun.rmi.transport.tcp.TCPChannel.newConnection(TCPChannel.java:204)
	at java.rmi/sun.rmi.server.UnicastRef.invoke(UnicastRef.java:134)
	at java.rmi/java.rmi.server.RemoteObjectInvocationHandler.invokeRemoteMethod(RemoteObjectInvocationHandler.java:215)
	at java.rmi/java.rmi.server.RemoteObjectInvocationHandler.invoke(RemoteObjectInvocationHandler.java:160)
	at jdk.proxy2/jdk.proxy2.$Proxy13.pullMessages(Unknown Source)
	at lsfusion.client.controller.remote.PingThread$1.call(PingThread.java:70)
	at lsfusion.client.controller.remote.PingThread$1.call(PingThread.java:1)
	at lsfusion.client.controller.remote.RmiQueue.runRetryableRequest(RmiQueue.java:148)
	at lsfusion.client.controller.remote.RmiQueue.runRetryableRequest(RmiQueue.java:101)
	at lsfusion.client.controller.remote.RmiQueue.runRetryableRequest(RmiQueue.java:97)
	at lsfusion.client.controller.remote.PingThread.run(PingThread.java:68)


19 окт. 2025 16:49:09,732 ERROR LogicsModule - java.net.ConnectException Connection refused: connect at '19.10.2025, 16:49' from 'dilpkn': 

	at java.base/sun.nio.ch.Net.connect0(Native Method)
	at java.base/sun.nio.ch.Net.connect(Net.java:579)
	at java.base/sun.nio.ch.Net.connect(Net.java:568)
	at java.base/sun.nio.ch.NioSocketImpl.connect(NioSocketImpl.java:585)
	at java.base/java.net.SocksSocketImpl.connect(SocksSocketImpl.java:327)
	at java.base/java.net.Socket.connect(Socket.java:633)
	at java.base/java.net.Socket.connect(Socket.java:583)
	at java.base/java.net.Socket.<init>(Socket.java:507)
	at java.base/java.net.Socket.<init>(Socket.java:287)
	at lsfusion.base.remote.ZipSocket.<init>(ZipSocket.java:26)
	at lsfusion.base.remote.ZipClientSocketFactory.createSocket(ZipClientSocketFactory.java:34)
	at java.rmi/sun.rmi.transport.tcp.TCPEndpoint.newSocket(TCPEndpoint.java:620)
	at java.rmi/sun.rmi.transport.tcp.TCPChannel.createConnection(TCPChannel.java:217)
	at java.rmi/sun.rmi.transport.tcp.TCPChannel.newConnection(TCPChannel.java:204)
	at java.rmi/sun.rmi.server.UnicastRef.invoke(UnicastRef.java:134)
	at java.rmi/java.rmi.server.RemoteObjectInvocationHandler.invokeRemoteMethod(RemoteObjectInvocationHandler.java:215)
	at java.rmi/java.rmi.server.RemoteObjectInvocationHandler.invoke(RemoteObjectInvocationHandler.java:160)
	at jdk.proxy2/jdk.proxy2.$Proxy13.pullMessages(Unknown Source)
	at lsfusion.client.controller.remote.PingThread$1.call(PingThread.java:70)
	at lsfusion.client.controller.remote.PingThread$1.call(PingThread.java:1)
	at lsfusion.client.controller.remote.RmiQueue.runRetryableRequest(RmiQueue.java:148)
	at lsfusion.client.controller.remote.RmiQueue.runRetryableRequest(RmiQueue.java:101)
	at lsfusion.client.controller.remote.RmiQueue.runRetryableRequest(RmiQueue.java:97)
	at lsfusion.client.controller.remote.PingThread.run(PingThread.java:68)


19 окт. 2025 16:49:09,762 ERROR LogicsModule - java.net.ConnectException Connection refused: connect at '19.10.2025, 16:49' from 'dilpkn': 

	at java.base/sun.nio.ch.Net.connect0(Native Method)
	at java.base/sun.nio.ch.Net.connect(Net.java:579)
	at java.base/sun.nio.ch.Net.connect(Net.java:568)
	at java.base/sun.nio.ch.NioSocketImpl.connect(NioSocketImpl.java:585)
	at java.base/java.net.SocksSocketImpl.connect(SocksSocketImpl.java:327)
	at java.base/java.net.Socket.connect(Socket.java:633)
	at java.base/java.net.Socket.connect(Socket.java:583)
	at java.base/java.net.Socket.<init>(Socket.java:507)
	at java.base/java.net.Socket.<init>(Socket.java:287)
	at lsfusion.base.remote.ZipSocket.<init>(ZipSocket.java:26)
	at lsfusion.base.remote.ZipClientSocketFactory.createSocket(ZipClientSocketFactory.java:34)
	at java.rmi/sun.rmi.transport.tcp.TCPEndpoint.newSocket(TCPEndpoint.java:620)
	at java.rmi/sun.rmi.transport.tcp.TCPChannel.createConnection(TCPChannel.java:217)
	at java.rmi/sun.rmi.transport.tcp.TCPChannel.newConnection(TCPChannel.java:204)
	at java.rmi/sun.rmi.server.UnicastRef.invoke(UnicastRef.java:134)
	at java.rmi/java.rmi.server.RemoteObjectInvocationHandler.invokeRemoteMethod(RemoteObjectInvocationHandler.java:215)
	at java.rmi/java.rmi.server.RemoteObjectInvocationHandler.invoke(RemoteObjectInvocationHandler.java:160)
	at jdk.proxy2/jdk.proxy2.$Proxy4.createNavigator(Unknown Source)
	at lsfusion.client.controller.remote.proxy.RemoteLogicsProxy.createNavigator_aroundBody0(RemoteLogicsProxy.java:28)
	at lsfusion.client.controller.remote.proxy.RemoteLogicsProxy.createNavigator_aroundBody1$advice(RemoteLogicsProxy.java:86)
	at lsfusion.client.controller.remote.proxy.RemoteLogicsProxy.createNavigator(RemoteLogicsProxy.java:1)
	at lsfusion.client.view.MainFrame$1$1.lambda$0(MainFrame.java:79)
	at lsfusion.interop.logics.AbstractLogicsProviderImpl.runRequest(AbstractLogicsProviderImpl.java:70)
	at lsfusion.interop.logics.AbstractLogicsProviderImpl.runRequest(AbstractLogicsProviderImpl.java:64)
	at lsfusion.client.controller.MainController.runRequest(MainController.java:446)
	at lsfusion.client.view.MainFrame$1$1.call(MainFrame.java:74)
	at lsfusion.client.view.MainFrame$1$1.call(MainFrame.java:1)
	at lsfusion.client.controller.remote.RmiQueue.runRetryableRequest(RmiQueue.java:148)
	at lsfusion.client.controller.remote.RmiQueue.runRetryableRequest(RmiQueue.java:101)
	at lsfusion.client.controller.remote.RmiQueue.runRetryableRequest(RmiQueue.java:97)
	at lsfusion.client.view.MainFrame$1.call(MainFrame.java:71)
	at lsfusion.client.view.MainFrame$1.call(MainFrame.java:1)
	at lsfusion.client.controller.remote.ReconnectWorker.doInBackground(ReconnectWorker.java:33)
	at java.desktop/javax.swing.SwingWorker$1.call(SwingWorker.java:304)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.desktop/javax.swing.SwingWorker.run(SwingWorker.java:343)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:833)


19 окт. 2025 16:49:12,145 ERROR SystemLogger - Failed to initialize tess4j. Add tess4j jar to classpath.
19 окт. 2025 17:00:25,306 ERROR SystemLogger - Failed to initialize tess4j. Add tess4j jar to classpath.
19 окт. 2025 17:21:37,611 ERROR LogicsModule - java.net.ConnectException Connection refused: connect at '19.10.2025, 17:21' from 'dilpkn': 

	at java.base/sun.nio.ch.Net.connect0(Native Method)
	at java.base/sun.nio.ch.Net.connect(Net.java:579)
	at java.base/sun.nio.ch.Net.connect(Net.java:568)
	at java.base/sun.nio.ch.NioSocketImpl.connect(NioSocketImpl.java:585)
	at java.base/java.net.SocksSocketImpl.connect(SocksSocketImpl.java:327)
	at java.base/java.net.Socket.connect(Socket.java:633)
	at java.base/java.net.Socket.connect(Socket.java:583)
	at java.base/java.net.Socket.<init>(Socket.java:507)
	at java.base/java.net.Socket.<init>(Socket.java:287)
	at lsfusion.base.remote.ZipSocket.<init>(ZipSocket.java:26)
	at lsfusion.base.remote.ZipClientSocketFactory.createSocket(ZipClientSocketFactory.java:34)
	at java.rmi/sun.rmi.transport.tcp.TCPEndpoint.newSocket(TCPEndpoint.java:620)
	at java.rmi/sun.rmi.transport.tcp.TCPChannel.createConnection(TCPChannel.java:217)
	at java.rmi/sun.rmi.transport.tcp.TCPChannel.newConnection(TCPChannel.java:204)
	at java.rmi/sun.rmi.server.UnicastRef.invoke(UnicastRef.java:134)
	at java.rmi/java.rmi.server.RemoteObjectInvocationHandler.invokeRemoteMethod(RemoteObjectInvocationHandler.java:215)
	at java.rmi/java.rmi.server.RemoteObjectInvocationHandler.invoke(RemoteObjectInvocationHandler.java:160)
	at jdk.proxy2/jdk.proxy2.$Proxy13.pullMessages(Unknown Source)
	at lsfusion.client.controller.remote.PingThread$1.call(PingThread.java:70)
	at lsfusion.client.controller.remote.PingThread$1.call(PingThread.java:1)
	at lsfusion.client.controller.remote.RmiQueue.runRetryableRequest(RmiQueue.java:148)
	at lsfusion.client.controller.remote.RmiQueue.runRetryableRequest(RmiQueue.java:101)
	at lsfusion.client.controller.remote.RmiQueue.runRetryableRequest(RmiQueue.java:97)
	at lsfusion.client.controller.remote.PingThread.run(PingThread.java:68)


19 окт. 2025 17:21:37,776 ERROR LogicsModule - java.net.ConnectException Connection refused: connect at '19.10.2025, 17:21' from 'dilpkn': 

	at java.base/sun.nio.ch.Net.connect0(Native Method)
	at java.base/sun.nio.ch.Net.connect(Net.java:579)
	at java.base/sun.nio.ch.Net.connect(Net.java:568)
	at java.base/sun.nio.ch.NioSocketImpl.connect(NioSocketImpl.java:585)
	at java.base/java.net.SocksSocketImpl.connect(SocksSocketImpl.java:327)
	at java.base/java.net.Socket.connect(Socket.java:633)
	at java.base/java.net.Socket.connect(Socket.java:583)
	at java.base/java.net.Socket.<init>(Socket.java:507)
	at java.base/java.net.Socket.<init>(Socket.java:287)
	at lsfusion.base.remote.ZipSocket.<init>(ZipSocket.java:26)
	at lsfusion.base.remote.ZipClientSocketFactory.createSocket(ZipClientSocketFactory.java:34)
	at java.rmi/sun.rmi.transport.tcp.TCPEndpoint.newSocket(TCPEndpoint.java:620)
	at java.rmi/sun.rmi.transport.tcp.TCPChannel.createConnection(TCPChannel.java:217)
	at java.rmi/sun.rmi.transport.tcp.TCPChannel.newConnection(TCPChannel.java:204)
	at java.rmi/sun.rmi.server.UnicastRef.invoke(UnicastRef.java:134)
	at java.rmi/java.rmi.server.RemoteObjectInvocationHandler.invokeRemoteMethod(RemoteObjectInvocationHandler.java:215)
	at java.rmi/java.rmi.server.RemoteObjectInvocationHandler.invoke(RemoteObjectInvocationHandler.java:160)
	at jdk.proxy2/jdk.proxy2.$Proxy13.pullMessages(Unknown Source)
	at lsfusion.client.controller.remote.PingThread$1.call(PingThread.java:70)
	at lsfusion.client.controller.remote.PingThread$1.call(PingThread.java:1)
	at lsfusion.client.controller.remote.RmiQueue.runRetryableRequest(RmiQueue.java:148)
	at lsfusion.client.controller.remote.RmiQueue.runRetryableRequest(RmiQueue.java:101)
	at lsfusion.client.controller.remote.RmiQueue.runRetryableRequest(RmiQueue.java:97)
	at lsfusion.client.controller.remote.PingThread.run(PingThread.java:68)


19 окт. 2025 17:21:37,869 ERROR LogicsModule - java.net.ConnectException Connection refused: connect at '19.10.2025, 17:21' from 'dilpkn': 

	at java.base/sun.nio.ch.Net.connect0(Native Method)
	at java.base/sun.nio.ch.Net.connect(Net.java:579)
	at java.base/sun.nio.ch.Net.connect(Net.java:568)
	at java.base/sun.nio.ch.NioSocketImpl.connect(NioSocketImpl.java:585)
	at java.base/java.net.SocksSocketImpl.connect(SocksSocketImpl.java:327)
	at java.base/java.net.Socket.connect(Socket.java:633)
	at java.base/java.net.Socket.connect(Socket.java:583)
	at java.base/java.net.Socket.<init>(Socket.java:507)
	at java.base/java.net.Socket.<init>(Socket.java:287)
	at lsfusion.base.remote.ZipSocket.<init>(ZipSocket.java:26)
	at lsfusion.base.remote.ZipClientSocketFactory.createSocket(ZipClientSocketFactory.java:34)
	at java.rmi/sun.rmi.transport.tcp.TCPEndpoint.newSocket(TCPEndpoint.java:620)
	at java.rmi/sun.rmi.transport.tcp.TCPChannel.createConnection(TCPChannel.java:217)
	at java.rmi/sun.rmi.transport.tcp.TCPChannel.newConnection(TCPChannel.java:204)
	at java.rmi/sun.rmi.server.UnicastRef.invoke(UnicastRef.java:134)
	at java.rmi/java.rmi.server.RemoteObjectInvocationHandler.invokeRemoteMethod(RemoteObjectInvocationHandler.java:215)
	at java.rmi/java.rmi.server.RemoteObjectInvocationHandler.invoke(RemoteObjectInvocationHandler.java:160)
	at jdk.proxy2/jdk.proxy2.$Proxy4.createNavigator(Unknown Source)
	at lsfusion.client.controller.remote.proxy.RemoteLogicsProxy.createNavigator_aroundBody0(RemoteLogicsProxy.java:28)
	at lsfusion.client.controller.remote.proxy.RemoteLogicsProxy.createNavigator_aroundBody1$advice(RemoteLogicsProxy.java:86)
	at lsfusion.client.controller.remote.proxy.RemoteLogicsProxy.createNavigator(RemoteLogicsProxy.java:1)
	at lsfusion.client.view.MainFrame$1$1.lambda$0(MainFrame.java:79)
	at lsfusion.interop.logics.AbstractLogicsProviderImpl.runRequest(AbstractLogicsProviderImpl.java:70)
	at lsfusion.interop.logics.AbstractLogicsProviderImpl.runRequest(AbstractLogicsProviderImpl.java:64)
	at lsfusion.client.controller.MainController.runRequest(MainController.java:446)
	at lsfusion.client.view.MainFrame$1$1.call(MainFrame.java:74)
	at lsfusion.client.view.MainFrame$1$1.call(MainFrame.java:1)
	at lsfusion.client.controller.remote.RmiQueue.runRetryableRequest(RmiQueue.java:148)
	at lsfusion.client.controller.remote.RmiQueue.runRetryableRequest(RmiQueue.java:101)
	at lsfusion.client.controller.remote.RmiQueue.runRetryableRequest(RmiQueue.java:97)
	at lsfusion.client.view.MainFrame$1.call(MainFrame.java:71)
	at lsfusion.client.view.MainFrame$1.call(MainFrame.java:1)
	at lsfusion.client.controller.remote.ReconnectWorker.doInBackground(ReconnectWorker.java:33)
	at java.desktop/javax.swing.SwingWorker$1.call(SwingWorker.java:304)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.desktop/javax.swing.SwingWorker.run(SwingWorker.java:343)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:833)


19 окт. 2025 17:21:44,221 ERROR SystemLogger - Failed to initialize tess4j. Add tess4j jar to classpath.
